### PR TITLE
added geekrepos in open source tools

### LINKED ISSUE #834: [Add] geekrepos in open source tools
### Site URL

https://geeksrepos.com/

### Description of the above provided site

This site will help you to get different open source project to contribute in. You will get open-source project repos from beginner to advanced level.

### Category

Open Source

### Sub Category

open-source-tools

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Bhanuteja01**
## Fixes Issue 
Closes #834

## Changes proposed
added geeksrepo in open source tools

### CODE REVIEW

File: database/opensource/tools.json [hunk 1]
```diff
@@ -27,4 +27,11 @@
         "category": "open source",
         "subcategory": "open-source-tools"
     }
```
Comment (rupali-codes) [inline database/opensource/tools.json:29]:
```suggestion
    },
```

**Review by rupali-codes — CHANGES_REQUESTED** [review]
hey, please a _comma_ in line 29 after closing the curly braces

**Comment by rupali-codes** [comment]
also, please update the description and add just _closes #834_ instead _closes issue #834_

**Comment by Bhanuteja01** [comment]
@rupali-codes 
can you tell me description to update in json or i need to mention  "updated the description" in PR template

File: database/opensource/tools.json [hunk 2]
```diff
@@ -26,5 +26,12 @@
         "url": "https://firstcontributions.github.io/",
         "category": "open source",
         "subcategory": "open-source-tools"
-    }
-]
\ No newline at end of file
+    },
+    {
+        "name": "Geeksrepos",
+        "description": "This site will help you to get different open source project to contribute in. You will get open-source project repos from beginner to advanced level.",
```
Comment (CBID2) [inline database/opensource/tools.json:32]: [Proofreading]
```suggestion
        "description": "Whether you're a beginner or experienced,this site will help you to find different open source projects to start your journey.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Bhanuteja01! I just commented on a proofreading error in my inline comment. In the future, consider checking the descriptions of your sources through a proofreading tool like Grammarly or spellcheck before making them your PRs.

**Review by CBID2 — DISMISSED** [review]
LGTM! :)

File: database/opensource/tools.json [hunk 3]
```diff
@@ -26,5 +26,12 @@
...
-]
\ No newline at end of file
+    },
+    {
+        "name": "Geeksrepos",
+        "description":"Whether you're a beginner or experienced,this site will help you to find different open source projects to start your journey.", 
+        "url": "https://geeksrepos.com/",
+        "category": "open source",
+        "subcategory": "open-source-tools"
+    },
```
Comment (rupali-codes) [inline database/opensource/tools.json:36]:
```suggestion
    }
```

**Review by rupali-codes — APPROVED** [review]
@Bhanuteja01 please be careful about adding and removing commas :)

**Comment by Bhanuteja01** [comment]
@rupali-codes   @CBID2 
Thanks for guiding me 
Next time I make sure thing to be right

**Comment by rupali-codes** [comment]
> @rupali-codes @CBID2 Thanks for guiding me Next time I make sure thing to be right

😊 sure, please let us know if you need any kinda help :)

**Pull request merged**